test/parallel/test-net-write-after-close.js

```diff
@@ -21,24 +21,27 @@
 
 'use strict';
 const common = require('../common');
+
 const net = require('net');
 
+let serverSocket;
+
 const server = net.createServer(common.mustCall(function(socket) {
+  serverSocket = socket;
+
   socket.resume();
 
   socket.on('error', common.mustCall(function(error) {
     console.error('got error, closing server', error);
     server.close();
```

Comment:
it seems this is the expected behavior. Can we change the text or place a comment so it's clear to the reader?